### ISSUE TITLE
Editing around +/- 180 east/west

### ISSUE THREAD

**Issue by RicoZ-OSM** [MarcusWolschon/osmeditor4android#389]
Fiji is suffering quite badly because many OSM tools and editors break here:
http://www.openstreetmap.org/?lat=-16.853&lon=-179.935&zoom=11&layers=M

Vespucci seems to work to some degree but still makes it necessary to pan all 
round the world to edit both sides of the small island Taveuni

Any chance to:
* download across +/-180
* pan across +/-180
?

**Comment by simonpoole** [comment]
Well it is "only a small matter of programming" ... and testing, given that this is notoriously easy to get wrong. But it is a low use edge case which is the reason why most editors don't bother. BTW soes JOSM work?

**Comment by RicoZ-OSM** [comment]
no, JOSM may actually seriously damage data when used in this area.

**Comment by simonpoole** [comment]
Well, no guarantees, but I'll have a look at it.

**Comment by RicoZ-OSM** [comment]
Thanks:)

**Comment by simonpoole** [comment]
- [ ] allow panning over date line
- [ ] split download bounding boxes
- [ ] figure out how to handle ways crossing the dateline (perhaps with two nodes at -180° and +180°?)

**Comment by jidanni** [comment]
Trouble also when panning in the Aleutians. Will probably need two bookmarks to make the jump. P.S., don't say dateline, say 180 deg.
